.github/workflows/dotnet-core.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
```

Comment:
cool, we can delete this file entirely, it has now been replaced by the one called `test.yaml` that you can see in the `exchange-api-client` repo for instance